### PR TITLE
en: Improve self-hosted k8s deployment

### PR DESCRIPTION
**Pull request by Joyinqin**
### What is changed, added or deleted? (Required)

Improve self-hosted k8s deployment.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:  https://github.com/pingcap/docs-tidb-operator/pull/804
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: en/configure-a-tidb-cluster.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ For the demonstration environment or functional verification, you can use networ
 
 > **Note:**
 >
-> If you set a storage class that does not exist in the TiDB cluster that you are creating, then the cluster creation goes to the Pending state. In this situation, you must [destroy the TiDB cluster in Kubernetes](destroy-a-tidb-cluster.md).
+> When you create the TiDB cluster and set a storage class that does not exist in the Kubernetes cluster, then the TiDB cluster creation goes to the Pending state. In this situation, you must [destroy the TiDB cluster in Kubernetes](destroy-a-tidb-cluster.md) and try again.
```
Comment (ran-huang) [inline en/configure-a-tidb-cluster.md:80]:
```suggestion
> When you create the TiDB cluster, if you set a storage class that does not exist in the Kubernetes cluster, then the TiDB cluster creation goes to the Pending state. In this situation, you must [destroy the TiDB cluster in Kubernetes](destroy-a-tidb-cluster.md) and retry the creation.
```

File: en/deploy-tidb-operator.md [hunk 2]
```diff
@@ -76,13 +74,15 @@ tidbinitializers.pingcap.com         2020-06-11T07:59:42Z
 tidbmonitors.pingcap.com             2020-06-11T07:59:41Z
 ```
 
-### Installation
+### Customize TiDB Operator deployment
+
+To deploy TiDB Operator quickly, you can refer to [Deploy TiDB Operator](get-started.md#deploy-tidb-operator). This section describes how to customize the deployment of TiDB Operator.
 
-After the various CRDs above are created, you can install TiDB Operator on your Kubernetes cluster. There are two installation methods: online and offline.
+After creating CRDs, there are two methods to deploy TiDB Operator on your Kubernetes cluster: online and offline.
```
Comment (ran-huang) [inline en/deploy-tidb-operator.md:81]:
```suggestion
After creating CRDs in the step above, there are two methods to deploy TiDB Operator on your Kubernetes cluster: online and offline.
```

File: en/deploy-tidb-operator.md [hunk 3]
```diff
@@ -76,13 +74,15 @@ tidbinitializers.pingcap.com         2020-06-11T07:59:42Z
...
 
-#### Online installation
+#### Online deployment
 
-1. Get the `values.yaml` file of the `tidb-operator` chart you want to install.
+1. Get the `values.yaml` file of the `tidb-operator` chart you want to deploy
```
Comment (ran-huang) [inline en/deploy-tidb-operator.md:85]:
```suggestion
1. Get the `values.yaml` file of the `tidb-operator` chart you want to deploy:
```
Please update other steps

File: en/prerequisites.md [hunk 4]
```diff
@@ -184,14 +188,15 @@ After the installation, take the following steps:
...
+
     {{< copyable "shell-regular" >}}
 
     ```shell
     vim /etc/systemd/system/docker.service
-    LimitNOFILE=1048576
     ```
 
-    Set `LimitNOFILE` as equal to or greater than `1048576`.
+    Set `LimitNOFILE` = `1048576`. Here, you can set `LimitNOFILE` to a number as equal to or greater than `1048576`.
```
Comment (ran-huang) [inline en/prerequisites.md:199]:
```suggestion
    Set `LimitNOFILE` to `1048576`. Here, you can set `LimitNOFILE` to a number equal to or greater than `1048576`.
```

**Comment by dragonly** [comment]
LGTM

**Review by ran-huang — APPROVED** [review]

**Pull request merged**